app/design/adminhtml/default/default/template/pinpay/form/pinpay.phtml

```diff
@@ -47,6 +47,19 @@ $_remoteAddr = Mage::helper('core/http')->getRemoteAddr(false);
         </div>
     </li>
 
+    <li style="display: none" class="online" id="c">
```

Comment:
@andihan Can you add an option to make this options in both the frontend and backend? Not all merchants will want to force their customers to enter the credit card type.